dm/master/server.go

```diff
@@ -23,6 +23,8 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/zap"
```

Comment:
it is better to re-organize import group based on DM custom regulation. ditto in some other files.

Reply:
I would re-organize all import path after we refactor log